cmd/http-tracer.go

```diff
@@ -199,5 +222,6 @@ func Trace(f http.HandlerFunc, logBody bool, w http.ResponseWriter, r *http.Requ
 
 	t.ReqInfo = rq
 	t.RespInfo = rs
+	t.CallStats = trace.CallStats{Latency: time.Since(startTime), InputBytes: reqBodyRecorder.Size(), OutputBytes: respBodyRecorder.Size()}
```

Comment:
Note that `Latency` can already be derived from `RespInfo.Time - ReqInfo.Time`. What is not already available though, is TTFB (time to first byte)

Reply:
Are you not approving the PR or is it just a suggestion @krishnasrinivas its unclear.